pyproject.toml

```diff
@@ -78,6 +78,7 @@ dependencies = [
 #
 all_extras = [
   "arch>=5.6,<6.4.0",
+  'autots>=0.6.10,<0.6.2',
```

Comment:
```suggestion
  'autots<0.7,>=0.6.2',
```

Reply:
I assume as well that they use semantic versioning, and with that 0.7 should be a valid upper bound. However, the current version (at least during implementation) was 0.6.1, so I suggest taking this as a lower bound.